app/src/main/res/drawable/edit_text_background_border_blue.xml

```diff
@@ -2,8 +2,8 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
   android:shape="rectangle">
   <corners android:radius="4dp" />
-  <solid android:color="@color/color_def_white" />
+  <solid android:color="@color/component_color_shared_text_input_layout_background_color" />
   <stroke
     android:width="2dp"
-    android:color="@color/color_def_oppia_dark_blue" />
+    android:color="@color/component_color_shared_input_interaction_edit_text_border_color" />
```

Comment:
Why are you naming it like 
`input_interaction_edit_text`
and above you are naming it as `text_input_layout_`

shouldn't these be consistent?

Reply:
> Why are you naming it like `input_interaction_edit_text` and above you are naming it as `text_input_layout_`
> 
> shouldn't these be consistent?

Correct these should be consistent but in `styles.xml`, for input interaction some are named as `...text_input...` and some as `..input_interaction...` so this was little confusion that which convention should be followed since for already existing color I have not created a new color and have used the existing one.

Reply:
In that case lets fix this confusion in this PR atleast for this file.

Reply:
> In that case lets fix this confusion in this PR atleast for this file.

I think `...input_interaction...` sounds better to me. 
 So should I replace `...text_input...` with `...input_interaction...`?